webforces/server/mongodbworker.py

```diff
@@ -101,6 +101,28 @@ def addUser(self, user) -> Tuple[DBStatus, User]:
         logger.debug("New user was successfully added")
         return (DBStatus.s_ok, user)
 
+    def updUser(self, user) -> DBStatus:
+        try:
+            users_collection = self.db["users"]
+            user_d = users_collection.find_one({"user_id": user.user_id})
+            if user_d is None:
+                logger.error("This user does not exist")
+                return DBStatus.s_data_issue
+            user_d = users_collection.update_one(
+                {"user_id": user.user_id}, {"$set": {
+                    "first_name": user.first_name,
+                    "second_name": user.second_name,
+                    "middle_name": user.middle_name,
+                }})
+            if user_d is None:
```

Comment:
I think it is an extra condtition.

`except Exception as e:` catches the fails of the pymongo methods.

Reply:
`if user_d is None` handles the case when there is no exception raised.